pytest_tap/plugin.py

```diff
@@ -51,6 +51,13 @@ def pytest_addoption(parser):
     )
 
 
+def pytest_runtestloop(session):
```

Comment:
I'd like this hook to go after the `pytest_configure` hook because it seems more logical there.